interfaces/Portalicious/src/app/domains/metric/metric.model.ts

```diff
@@ -3,3 +3,10 @@ import { ProgramStats } from '@121-service/src/metrics/dto/program-stats.dto';
 import { Dto } from '~/utils/dto-type';
 // TODO: AB#30152 This type should be refactored to use Dto121Service
 export type ProjectMetrics = Dto<ProgramStats>;
+
+export interface SummaryMetric {
+  value: null | number | string | undefined;
+  label: string;
+  type?: string;
```

Comment:
There is currently not use for this property, right?
Or is there already an implementations somewhere that uses this for switching between `number`s/`Date`s/etc?